driver/Makefile

```diff
@@ -116,7 +116,7 @@ DEBUGGER :=
 COMPILER_FLAGS =
 
 #	Specify any additional linker flags to be used.
-LINKER_FLAGS =
+LINKER_FLAGS = -lstdc++
```

Comment:
see https://github.com/haiku/haiku/blob/master/build/jam/KernelRules#L153 -nostdlib is included.

Reply:
I know the option  -fno-rtti is often used for drivers using c++

Reply:
Thanks korli! I'll have another look once I find the time! The peculiarities of the compiler have been removed from my brain thanks to working with Java for the last years ;-)

Reply:
Okay, I now got to the point where the build fails again, but at a different point :-)

I now get undefined references for `operator delete(void*, std::nothrow_t const&)` and `__cxa_throw_bad_array_new_length`.

The former might be caused because we don't seem to add this variant in kernel_cpp.cpp. The latter, I don't really know why; all 'new' are using (std::nothrow).

Reply:
__cxa_throw_bad_array_new_length seems to be linked when using new[]. Looking at https://gcc.gnu.org/bugzilla/show_bug.cgi?id=71757#c0 it seems the compiler is doing this with newer std. Using -std=gnu++98 would avoid this.

Anyway, you can add a definition like this as a workaround.
extern "C" void __attribute__((weak)) __cxa_throw_bad_array_new_length() {}